src/main/java/Yappatron.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class Yappatron {
+    public static final int MAX_TASKS = 100;
+    public static int taskNumber = 0;
+    public static Task[] taskArray = new Task[MAX_TASKS];
```

Comment:
The use of a fixed-size array (Task[]) reduces flexibility and scalability. Exceeding MAX_TASKS will cause the program to fail, limiting the number of tasks that can be managed.